lib/cucumber/events/after_test_step.rb

```diff
@@ -0,0 +1,30 @@
+module Cucumber
+  module Events
+
+    # Event fired after each test step has been executed
+    class AfterTestStep
```

Comment:
@mattwynne Is that really an event or a name for a hook? Normally I would say, that `TestStepHasRun` would be a name for an event.